Assets/Scripts/Layers/LayerTypes/CartesianTileBuildingColorLayerGameObject.cs

```diff
@@ -1,16 +1,56 @@
+using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using Netherlands3D.SubObjects;
+using Netherlands3D.Twin.DataSets;
+using Netherlands3D.Twin.Layers.Properties;
+using UnityEngine;
+using UnityEngine.Events;
 
 namespace Netherlands3D.Twin.Layers
 {
-    public class DatasetLayerGameObject : LayerGameObject
+    public class CartesianTileBuildingColorLayerGameObject : LayerGameObject, ILayerWithPropertyData
```

Comment:
het is nu specifieker dan het was, je kon ook terrein kleuren omdat die ook subobject ids had

Reply:
misschien CartesianTileColorLayerGameObject van maken

Reply:
In overleg veranderd naar CartesianTileSubObjectColor*